components/mailer.rst

```diff
@@ -177,6 +177,43 @@ the application or when using a self-signed certificate::
 
     The ``verify_peer`` option was introduced in Symfony 5.1.
 
+Other options
+-------------
+
+.. versionadded:: 5.2
+
+    These options were added in Symfony 5.2.
```

Comment:
This could be problematic if further options will be added, lets say in `5.3`. I would recommend to remove this versionadded directive and do it like...